test/unit/decorators/rpc.decorator.unit.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
```

Comment:
grpc?

Reply:
![image](https://user-images.githubusercontent.com/1533239/32349962-d6049186-bfe6-11e7-9cc6-9e1617709785.png)

@raymondfeng thanks for reviewing, I'm using rpc for semantics.

What we are actually implementing are RPC Methods (gRPC stands for the entire RPC System), IMO following semantics would help understand easier people reading the GRPC documentation.

For instance, the entire module is named grpc.component which attempts to achieve the entire gRPC functionality, rpc stands for RPC methods.

Thoughts?